<launch>
    <rosparam param="serial_port">/dev/ttyACM0</rosparam>
    <rosparam command="load" file="$(find decision)/param/decision.yaml" />
    <node pkg="decision" type="serial_test_receive_node" name="serial_test_receive_node"/>
    <!-- <node pkg="decision" type="decision_node" name="decision_node" output="screen" /> -->
</launch>
